packages/apps/reputation-oracle/server/src/common/enums/nda.ts

```diff
@@ -0,0 +1,4 @@
+export enum NdaStatus {
+  SIGNED = 'SIGNED',
+  REJECTED = 'REJECTED',
```

Comment:
Regenerate migrations after this
```suggestion
  SIGNED = 'signed',
  REJECTED = 'rejected',
```